middleware/markdown/markdown_tpl.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>{{.Title}}</title>
+</head>
+<body>
+{{.Include "header.html"}}
+Welcome to {{.Var.sitename}}!
```

Comment:
Thanks! We can, and I think we already talked about it, but then we have either to use `.context.Include` or create a custom type of `map[string]interface{}` that has the same methods as `middleware.Context` to be consistant with the other templates in the `browse`and `templates` directives.